uw-frame-components/portal/widgets/controllers.js

```diff
@@ -400,6 +400,98 @@ define(['angular'], function(angular) {
     initializeActionItems();
   }])
 
+  // VARIABLE CONTENT widget type
+  .controller('VariableContentController', [
+    '$scope', '$filter', '$log', function($scope, $filter, $log) {
+    /**
+     * Display provided custom template
+     *   - provisional: Check if it's the annual benefits enrollment
+     *   widget and display markup specific to that widget instead
+     * @param template The provided custom angular/HTML template
+     * @param start The day/time this content became active
+     * @param end The date/time this content will go away
+     */
+    var displayVariableContent = function(template, start, end) {
+      $scope.startDate = start;
+      $scope.endDate = end;
+      if ($scope.widget.fname === 'university-staff-benefits-statement') {
+        configureAnnualBenefitsEnrollmentContent();
+      } else {
+        $scope.template = template;
+      }
+    };
+
+    /**
+     *
+     */
+    var configureAnnualBenefitsEnrollmentContent = function() {
+      // Set enrollment period dates for current year
+      $scope.enrollStartDate = (new Date).getFullYear() + '-'
```

Comment:
:notebook: :clock1: https://momentjs.com/docs/#/displaying/format/

Reply:
I would LOVE to use moment. Hopefully this gets the conversation started again :)